app/views/hackathon_mailer/swag_request.text.erb

```diff
@@ -0,0 +1,5 @@
+Hey mail team! 👋
+
+Hackathon swag has been requested for:
+
+<%= @hackathon.swag_request.mailing_address.to_s %>
```

Comment:
Please include the name of the Hackathon, and the applicant's name (if it is present). Ideally, we also format the address as multiline.

See https://github.com/hackclub/hackathons-backend/issues/363#issuecomment-1928512159